cmd/lakectl/cmd/runs_show.go

```diff
@@ -0,0 +1,104 @@
+package cmd
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/go-openapi/swag"
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/api/gen/models"
+	"github.com/treeverse/lakefs/cmdutils"
+	"github.com/treeverse/lakefs/uri"
+)
+
+const actionRunResultTemplate = `
+Run ID: {{ .RunID }}
+	Event: {{ .EventType }}
+	Branch: {{ .Branch }}
+	Start time: {{ .StartTime }}
+	End time: {{ .EndTime }}
+{{if .CommitID}}	Commit ID: {{ .CommitID }}{{end}}
+	Status: {{ .Status }}
+`
+
+const actionTaskResultTemplate = `{{ $log := .HookLog }}
+{{ range $val := .Hooks }}
+Hook Run ID: {{ $val.HookRunID }}
+	Hook ID: {{ $val.HookID }}
+	Start time: {{ $val.StartTime }}
+	End time: {{ $val.EndTime }}
+	Action: {{ $val.Action }}
+	Status: {{ $val.Status }}
+Output:
+{{ printf $val.HookRunID | call $log }}
+{{ end }}{{ .Pagination | paginate }}
+`
+
+const runsShowRequiredArgs = 2
+
+var runsShowCmd = &cobra.Command{
+	Use:     "show",
```

Comment:
`show` is too general IMO. Maybe `describe`?

Reply:
wanted something like the current show command, `show         See detailed information about an entity by ID (commit, user, etc)`.